### ISSUE TITLE
Cannot use hardware wallet on desktop when browser window is small

### ISSUE THREAD

**Issue by ynx0** [urbit/bridge#547]
When on a desktop, a user cannot use a hardware wallet to authenticate if their browser is too small, presumably because the detection is done solely based on browser window dimensions rather than user agent.

![urbit bridge bug](https://user-images.githubusercontent.com/9275716/104860469-cb8c4280-58f9-11eb-8b29-19eeb8be1050.gif)

A cursory glance at the code reveals these points of interest:

* https://github.com/urbit/bridge/blob/a2dd484e3a5f0f1a24e51bd80c5c1077a3b03f1c/src/views/Login/Other.js#L48-L57

* https://github.com/urbit/bridge/blob/a2dd484e3a5f0f1a24e51bd80c5c1077a3b03f1c/src/lib/useBreakpoints.js#L1-L5

* https://github.com/urbit/bridge/blob/a2dd484e3a5f0f1a24e51bd80c5c1077a3b03f1c/src/lib/MediaQuery.js#L7-L11

* https://github.com/urbit/bridge/blob/a2dd484e3a5f0f1a24e51bd80c5c1077a3b03f1c/src/lib/useMedia.js

the last of which is based on https://usehooks.com/useMedia/ , which is based on screen size.

[`react-device-detect`][1] is a possible library to replace this functionality that wraps detection functionality based on user-agent and some other browser metadata.

[1]: https://www.npmjs.com/package/react-device-detect

**Comment by Fang-** [comment]
Good catch! Wondering why we marked those as unsupported on mobile, in any case. Wouldn't be too surprised if those are supported by some mobile browsers now...
